api/v1/validatorstate.go

```diff
@@ -94,8 +94,12 @@ func (v *ValidatorState) UnmarshalJSON(input []byte) error {
 	return err
 }
 
-func (v ValidatorState) String() string {
-	return validatorStateStrings[v]
+func (v ValidatorState) String() (string, error) {
+	if int(v) < 0 || int(v) >= len(validatorStateStrings) {
```

Comment:
The first recast isn't necessary; the compiler will convert the hardcoded 0 to a ValidatorState.